device.mk

```diff
@@ -205,4 +205,26 @@ PRODUCT_PROPERTY_OVERRIDES += \
 #
 PRODUCT_DEFAULT_PROPERTY_OVERRIDES += \
     persist.sys.usb.config=mtp \
```

Comment:
It may also be that mass_storage by itself is insufficient. The CAF stuff seems to often use great long combinations of diagnotics and other things, and the the string storef in persist.sys.usb.config (which gets stored in sys.usb.config) needs to match one of the strings shown init.qcom.usb.rc. It needs to match exactly using a simple string comparison.